src/Features/Core/Portable/CodeRefactorings/AddMissingImports/AddMissingImportsAnalysisResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.AddImport;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.AddMissingImports
+{
+    internal sealed class AddMissingImportsAnalysisResult
+    {
+        public ImmutableArray<AddImportFixData> AddImportFixData { get; }
+        public Document Document { get; }
```

Comment:
as per above.  this should just go away.

given that, i think we don't need this type at all.  anyone returning one of tehse can just return the array of fix-data.

Reply:
Please try to link to the comment in the future. `Above` doesn't mean a lot, especially considering the `above` comment for me was https://github.com/dotnet/roslyn/pull/48501/files#r512334438 

I think you're referring to https://github.com/dotnet/roslyn/pull/48501/files#r512333936

Reply:
Let's use https://github.com/dotnet/roslyn/pull/48501/files#r512333936 to drive this discussion